driver-adapters/planetscale-cf-basic/index.test.js

```diff
@@ -0,0 +1,126 @@
+// @ts-check
+const { test, expect } = require('@jest/globals')
+const fetch = require('node-fetch')
+const fs = require('fs')
+
+const pjson = require('./package.json')
+
+jest.setTimeout(30000)
+
+test.skip('prisma version and output', async () => {
```

Comment:
This is skipped?

Reply:
Yes, it will stay skipped until we have a working wasm engine and instead has a test that captures the full error below this one. (ps: I don't ever use `.failing` because it is unreliable with the bugs it has).

Reply:
Maybe create an issue to unskip later then? and add it as a comment?

Reply:
Once the test passes (ie. once wasm engine is implemented), the current snapshot will fail (as not an error anymore) so we will know and I then plan to create a quick PR to unskip it.